Tests/Common/Packets/BacktestNodePacketTests.cs

```diff
@@ -250,7 +250,6 @@ internal class TestInitialCashAmountAlgorithm : BasicTemplateDailyAlgorithm
         {
             public override void Initialize()
             {
-                SetAccountCurrency("EUR");
```

Comment:
The idea here is that the backtest job packet overrides the algorithms account currency which gets ignored, believe this is why we were directly returning at the `SecurityPortfolioManager` when already set, kinda of the inverse behavior that this PR wants to achieve. Let's discuss offline but maybe the check being removed, if still required, could potentially be applied only for backtesting at the algorithm layer 🤔